app/src/main/java/com/example/paetz/yacguide/utils/WidgetUtils.java

```diff
@@ -78,4 +78,16 @@ private static TextView _createTextView(Context context, String text, int textSi
 
         return textView;
     }
+
+    public static String getAscendSymbol(int styleId) {
+        String symbol = " ";
+        if (styleId < 9) {
```

Comment:
We should not operate on ids since this is error-prone and not easy to follow. Please create an enum.

Reply:
Sure, makes sense.

Reply:
Best would be a separate class AscendStyles:
```suggestion
      public class AscendStyles {
           private class AscendStyle { 
                 int id;
                 String name;
                 String symbol;
             }
              // init static ascendStyle list here
             
              public List<String> getAscendNames() {
                   // return all names as string list
              } 
              public int getAscendId(String name) { ... }
              
              public String getAscendSymbol(int id) { ... }
         }
```

Reply:
That approach would also work. I went for a simpler variant, but can change it, if you prefer.